ecl/hql/hqlgram2.cpp

```diff
@@ -3333,6 +3352,79 @@ IHqlExpression *HqlGram::lookupSymbol(IHqlScope * scope, IIdAtom * searchName)
     return scope->lookupSymbol(searchName, LSFpublic, lookupCtx);
 }
 
+const char * HqlGram::composeSymbolSuggestionMsg(IHqlExpression * symbol, StringBuffer & msg)
+{
+    if (symbol)
+        msg.append(" - did you perhaps mean \"").append(symbol->queryName()->queryStr()).append("\"?");
+    return msg.str();
+}
+
+IHqlExpression * HqlGram::findNearestSymbol(unsigned & editDistance, const HqlExprArray & symbolArray, const unsigned * distanceArray, unsigned penalty)
```

Comment:
This function doesn't belong in this class.  I don't think it uses any of the class members.